examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/activities/GrowBananaActivity.java

```diff
@@ -26,11 +26,13 @@ public record GrowBananaActivity(int quantity, Duration growingDuration) {
   }
 
   @Validation("Quantity must be positive")
+  @Validation.Subject({"quantity"})
```

Comment:
You can drop the containing curly braces if there's only one element. Very ergonomic :)

Reply:
Woah! Cool! Thanks – done :)